odoo/addons/base/tests/test_ir_actions.py

```diff
@@ -736,7 +736,7 @@ def test_related_field(self):
 
         # create a non-computed field, and assert how many queries it takes
         model_id = self.env['ir.model']._get_id('res.partner')
-        query_count = 48
+        query_count = 49
```

Comment:
What is the additional query?

Reply:
IIANM it's caused by the `format.vat.label.mixin`